spec/honeycomb/integrations/rack_spec.rb

```diff
@@ -51,7 +51,7 @@ def last_name
       end
 
       def created_at
-        Time.now
+        Time.new 2002, 3, 4, 5, 6, 7
```

Comment:
what was this fixing?

Reply:
No specific issue, just a general aversion to having variable dates in tests; I've fixed too many tests that flaked out due to time quirks. I can back it out if you want.